akka-contrib/src/main/scala/akka/contrib/pattern/ClusterClient.scala

```diff
@@ -311,10 +332,14 @@ object ClusterReceptionist {
    */
   private[pattern] object Internal {
     @SerialVersionUID(1L)
-    case object GetContacts
+    case object GetContacts extends DeadLetterSuppression
     @SerialVersionUID(1L)
     case class Contacts(contactPoints: immutable.IndexedSeq[ActorSelection])
     @SerialVersionUID(1L)
+    case object Heartbeat extends DeadLetterSuppression
+    @SerialVersionUID(1L)
+    case object HeartbeatRsp extends DeadLetterSuppression
```

Comment:
Regarding the 2.3.12 vs 2.3.13 compatibility, we could support 2.3.12 client with 2.3.13 server if we use `Identify` and `ActorIdentity` instead of these new heartbeat messages. WDYT?